### ISSUE TITLE
Set up Sorry Cypress and connect our Cypress tests to it

### ISSUE THREAD

**Issue by cweagans** [department-of-veterans-affairs/va.gov-cms#8848]
## Background

Before we can parallelize Cypress tests, we need to implement an open source dashboard to enable this functionality.
Ref: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8662

## Acceptance Criteria

- [ ] Ensure ECS with Fargate successfully runs Sorry Cypress and integrates with existing testing pipeline
- [ ] Update infrastructure documentation as needed

## Implementation steps

* One approach we kicked around was to just run https://github.com/sorry-cypress/sorry-cypress/blob/master/docker-compose.full.yml on the ECS cluster. We'd need to figure out DNS + an s3 bucket + keys for that bucket.

* If for whatever reason the s3 bucket and keys becomes a problem, we could also use the minio version: https://github.com/sorry-cypress/sorry-cypress/blob/master/docker-compose.minio.yml

### CMS Team
Please check the team(s) that will do this work.

- [x] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by cweagans** [comment]
ECS + Docker Compose: https://aws.amazon.com/blogs/containers/deploy-applications-on-amazon-ecs-using-docker-compose/

**Comment by olivereri** [comment]
It was a bit difficult to take Sorry Cypress' Docker Compose file and load it in to Fargate. They do provide a Cloudformation Stack where all of the related AWS Fargate configurations can be found. After getting the Task Definition to successfully run and creating a Service the next challenge is providing stable and appropriate access.

To prove the dashboard is accessible and the App at least runs I manually modified the Utility Tools ALB. The way the App is configured it requires a default rule that points to the dashboard. Not idea of a shared ALB. I might be overlooking something but I don't see a config to alter that and send it down a different path, e.g.  `/sorry-cypress-dashboard` instead of `/`. Could be an after deployment config.

**Comment by olivereri** [comment]
After getting this up and running on ECS Fargate I want to try and get this up on the Platform's EKS cluster. Convert the Compose file to K8s resources. This should make it more flexible as the underlying K8s should be configured better than ECS Fargate to handle working in the VA Network.

**Comment by olivereri** [comment]
Getting Sorry Cypress deployed on Platform Ops K8s isn't 100% straight forward and is fairly involved. Despite this I think it's right direction to go instead of rolling our own with ECS/Fargate if we intend to use this long term. Additionally, getting the experience of deploying apps that we'll use on K8s with ArgoCD will serve us well in the future.

There is a bullet list of requirements/tasks so we're not starting from scratch.
https://vfs.atlassian.net/wiki/spaces/OT/pages/1474594142/Onboard+a+New+Application+to+the+EKS+Infrastructure+As+a+VSP+team

Demian also added some specificity to that bullet list for what we're trying to do:
https://dsva.slack.com/archives/CBU0KDSB1/p1651618623228929?thread_ts=1651613951.169729&cid=CBU0KDSB1

Full Platform Ops discussion here:
https://dsva.slack.com/archives/CBU0KDSB1/p1651613951169729

**Comment by olivereri** [comment]
@jkalexander7 @ENeal49 I no longer think this'll be completed in the current sprint if we do it the _right way_, which is in the Platform ecosystem. I can possibly make an MVP (ECS/Fargate like originally planned) that will unblock @ndouglas work before Friday. However, I'd still want to put this on the K8s cluster and tear down the MVP which just adds work. Would like input because both directions I think are viable.

**Comment by ndouglas** [comment]
I don't think I'm particularly close to wrapping up #8518, so don't feel any pressure to unblock #8662 IMHO.  FWIW, I think if this is worth doing, it's worth doing for the long run.  We'd definitely all benefit from gaining more familiarity with EKS too.

**Comment by olivereri** [comment]
Would like to repurpose this as a Spike since it lead to the creation of #9007 and #8962.

**Comment by olivereri** [comment]
closing this in favor #8962
